kasa/smartstrip.py

```diff
@@ -93,8 +93,22 @@ async def update(self, update_children: bool = True):
         Needed for methods that are decorated with `requires_update`.
         """
         await super().update(update_children)
+        self.create_children()
+        if update_children and self.has_emeter:
+            for plug in self.children:
+                await plug.update()
+
+    def update_from_discovery_info(self, info):
+        """Update state from info from the discover call."""
+        super().update_from_discovery_info(info)
+        self.create_children()
 
-        # Initialize the child devices during the first update.
+    @requires_update
+    def create_children(self):
```

Comment:
```suggestion
    def _create_children(self):
```
There should be no need for any lib user to call this manually, so let's make it private.